scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -85,15 +85,16 @@ ENV PATH="/home/sof/work/xtensa-byt-elf/bin:${PATH}"
 ENV PATH="/home/sof/work/xtensa-hsw-elf/bin:${PATH}"
 ENV PATH="/home/sof/work/xtensa-apl-elf/bin:${PATH}"
 ENV PATH="/home/sof/work/xtensa-cnl-elf/bin:${PATH}"
+ENV PATH="/home/sof/work/xtensa-imx-elf/bin:${PATH}"
 
 RUN cd /home/sof && git clone https://github.com/jcmvbkbc/newlib-xtensa.git newlib-xtensa.git && \
 	cd newlib-xtensa.git && git checkout -b xtensa origin/xtensa && \
-	for arch in byt hsw apl cnl; do \
+	for arch in byt hsw apl cnl imx; do \
 		./configure --target=xtensa-${arch}-elf \
 		--prefix=/home/sof/work/xtensa-root && \
 		make && \
 		make install && \
-		make distclean; \
+		rm -rf etc/config.cache; \
```

Comment:
If it is not a big trouble I will move this to a separate patch, because replacing "rm -rf .." with make distclean it is a different change and should be pushed separately.

Reply:
@dbaluta @jajanusz 
Wow, that is some little optimization to speed up the build.
Just keep it like this they are almost same. I just got align with our wiki.